src/main/errors/Error.ts

```diff
@@ -0,0 +1,24 @@
+import { ErrorMessages } from './ErrorMessages';
+
+class BaseError extends Error {
+  public name: string;
+  public status: number;
+
+  constructor(errorName: string, errorMessage: string, status: number) {
+    super(errorMessage);
+    this.name = errorName;
+    this.status = status;
+  }
+}
+
+class ForbiddenError extends BaseError {
+  constructor(errorMessage?: string) {
+    super(
+      'ForbiddenError',
```

Comment:
Minor thing, but what do you think about an `ErrorCodes` class, so we could do something like `ErrorCodes.FORBIDDEN_ERROR`, similar to `ErrorMessages`. Totally okay if you think its too much

Reply:
Hmm, I think it's okay for now but I think we could def consider it for the future when we have more error types and whatnot! :)